### PR TITLE
fix duplicate component label in jaeger component

### PR DESCRIPTION
**Pull request by uwedeportivo**
label was declared twice and YAML decoders complain (but not k8s weirdly enough)

### CODE REVIEW

File: base/jaeger/jaeger.Deployment.yaml [hunk 1]
```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/component: jaeger
     app: jaeger
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/component: all-in-one
```
Comment (ggilmore) [inline base/jaeger/jaeger.Deployment.yaml:11]: @uwedeportivo Are you sure that jaeger isn't relying on this?

**Review by ggilmore — APPROVED** [review]

File: base/jaeger/jaeger.Deployment.yaml (hunk 1 above)
Comment (uwedeportivo) [reply base/jaeger/jaeger.Deployment.yaml:11]: i'm not sure. i googled a little bit and didn't find anything. the jaeger k8s operator might use it. not sure

Comment (ggilmore) [reply base/jaeger/jaeger.Deployment.yaml:11]: Okay. We should test to make sure that this change is safe .

Comment (uwedeportivo) [reply base/jaeger/jaeger.Deployment.yaml:11]: i tested with a test cluster and it seems fine. jaeger pods came up, i could hit the UI

**Pull request merged**